base/src/org/adempiere/process/InvoiceGenerateFromShipmentAbstract.java

```diff
@@ -1,173 +0,0 @@
-/******************************************************************************
```

Comment:
This class is automatically generated from ADempiere is better use InvoiceGenerateFromShipment.java class